samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/listener/delivery/LinearAccessDeliveryActionListener.java

```diff
@@ -194,7 +194,7 @@ public void processAction(ActionEvent ae) throws
           }
           eventLogData.setSiteId(site_id);
           
-          eventLogData.setErrorMsg(eventLogMessages.getString("no_submission"));
+          eventLogData.setErrorMsg("no_submission_user_no_submit");
```

Comment:
I think you could remove the eventLogMessages and the ResourceBundle import in this file like you do in the other ones.